gnomad_qc/v4/annotations/generate_freq.py

```diff
@@ -899,20 +889,55 @@ def create_final_freq_ht(ht: hl.Table) -> hl.Table:
     """
     Create final freq Table with only desired annotations.
 
-    Drop the following annotations:
-        - 'age_high_ab_hist', all annotations from the split VDSs....
-
-    Rename the following annotations:
-        - 'ab_adjusted_freq' -> 'freq'
-        - decide if we want to store uncorrected? Probably,just rename it.
+    .. note::
 
-    Filter the following from the 'freq' field:
-        - gatk versions
+        `pre_adjustment_freq` is the frequency annotation before any high AB het
+        adjustment. This value will be the same as freq for sites that have an AF below
+        the AF adjustment threshold.
 
     :param ht: Hail Table containing all annotations.
-    :return: Hail Table with only desired annotations.
+    :return: Hail Table with final annotations.
     """
-    raise NotImplementedError("Creation of final freq Table is not implemented yet.")
+    logger.info("Dropping gatk_version from freq ht array annotations...")
+    freq_meta, array_exprs = filter_arrays_by_meta(
+        ht.freq_meta,
+        {
+            "freq": ht.ab_adjusted_freq,
+            "freq_meta_sample_count": ht.index_globals().freq_meta_sample_count,
+            "high_ab_hets_by_group": ht.high_ab_hets_by_group,
+            "pre_adjustment_freq": ht.freq,
+        },
+        items_to_filter=["gatk_version"],
+        keep=False,
+    )
+
+    ht = ht.select(
+        freq=array_exprs["freq"],
+        faf=ht.faf,
+        grpmax=ht.grpmax,
```

Comment:
Just confirming that we are not changing `pop` here:
```
    'grpmax': struct {
        gnomad: struct {
            AC: int32, 
            AF: float64, 
            AN: int32, 
            homozygote_count: int64, 
            pop: str, 
            faf95: float64
        }, 
        non_ukb: struct {
            AC: int32, 
            AF: float64, 
            AN: int32, 
            homozygote_count: int64, 
            pop: str, 
            faf95: float64
        }
    } 
```
Can you ping in the gnomad channel about changing `pop` -> `ga_grp` and maybe make a poll with a few that we came up with as options?

Reply:
I wasn't going to originally but I think this is the only place I can think of that will have "pop" in the main release HT outside of the globals so now I'm leaning towards just changing it here. I polled the group here and `gen_anc` won.